security/sec-iride/src/main/java/org/geoserver/security/iride/IrideUserGroupService.java

```diff
@@ -0,0 +1,183 @@
+        // TODO: check if needed and, if so, implement
+        return super.getGroupByGroupname(groupname);
+    }
+
+    /*
+     *
+     * (non-Javadoc)
+     * @see org.geoserver.security.impl.AbstractUserGroupService#getUserByUsername(java.lang.String)
+     */
+    @Override
+    public GeoServerUser getUserByUsername(String username) throws IOException {
+        // TODO: check if needed and, if so, implement
+        return super.getUserByUsername(username);
+    }
+
+    /*
+     * (non-Javadoc)
+     * @see org.geoserver.security.impl.AbstractUserGroupService#createUserObject(java.lang.String, java.lang.String, boolean)
+     */
+    @Override
+    public GeoServerUser createUserObject(String username, String password, boolean isEnabled) throws IOException {
+        // TODO: check if needed and, if so, implement
+        return super.createUserObject(username, password, isEnabled);
+    }
+
+    /*
+     * (non-Javadoc)
+     * @see org.geoserver.security.impl.AbstractUserGroupService#createGroupObject(java.lang.String, boolean)
+     */
+    @Override
+    public GeoServerUserGroup createGroupObject(String groupname, boolean isEnabled) throws IOException {
+        // TODO: check if needed and, if so, implement
+        return super.createGroupObject(groupname, isEnabled);
+    }
+
+    /*
+     * (non-Javadoc)
+     * @see org.geoserver.security.impl.AbstractUserGroupService#getUsers()
+     */
+    @Override
+    public SortedSet<GeoServerUser> getUsers() throws IOException {
+        // TODO: check if needed and, if so, implement
+        return super.getUsers();
+    }
+
+    /*
+     * (non-Javadoc)
+     * @see org.geoserver.security.impl.AbstractUserGroupService#getUserGroups()
+     */
+    @Override
+    public SortedSet<GeoServerUserGroup> getUserGroups() throws IOException {
+        // TODO: check if needed and, if so, implement
+        return super.getUserGroups();
+    }
+
+    /*
+     * (non-Javadoc)
+     * @see org.geoserver.security.impl.AbstractUserGroupService#getUsersForGroup(org.geoserver.security.impl.GeoServerUserGroup)
+     */
+    @Override
+    public SortedSet<GeoServerUser> getUsersForGroup(GeoServerUserGroup group) throws IOException {
+        // TODO: check if needed and, if so, implement
+        return super.getUsersForGroup(group);
+    }
+
+    /*
+     * (non-Javadoc)
+     * @see org.geoserver.security.impl.AbstractUserGroupService#getGroupsForUser(org.geoserver.security.impl.GeoServerUser)
+     */
+    @Override
+    public SortedSet<GeoServerUserGroup> getGroupsForUser(GeoServerUser user) throws IOException {
+        // TODO: check if needed and, if so, implement
+        return super.getGroupsForUser(user);
+    }
+
+    /* (non-Javadoc)
+     * @see org.geoserver.security.impl.AbstractUserGroupService#deserialize()
+     */
+    @Override
+    protected void deserialize() throws IOException {
+        /* NOP */
+    }
+
```

Comment:
As a general notice, there are a lot of empty overrides. This is confusing. You should avoid put superfluous code in a Pull Request.